locales/en-US/notes.properties

```diff
@@ -31,7 +31,11 @@ disableSync=Disable Sync
 # LOCALIZATION NOTE (syncComplete): {date} is the date of last sync. If this
 # structure doesn't work for your locale, you can translate this as "Last sync:
 # {date}".
-syncComplete=Synced {date}
+syncComplete=Synced at {date}
```

Comment:
Unwanted change? In case, you need to change the string ID to change an existing string
https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Changing_existing_strings

Don't forget to update the reference in the comment.